src/main/java/io/vlingo/http/RequestParser.java

```diff
@@ -257,7 +257,7 @@ private void parseBody() {
 
     private void parseHeaders() {
       if (!continuation) {
-        headers = new Headers<>(2);
+        headers = new Headers<>(8);
```

Comment:
@VaughnVernon 8 is enough for our test, but do you think it is a realistic initial size to begin with in practice?

Reply:
@buritos One header `Host` is always required, and there will often be `Content-Length`. That's why I chose two by default. I don't have a good feel for how many headers on average are used in an application or service. If they are using content negotiation then `Content-Type` is used, and `Bearer-Token` is common for security. We use `X-Correlation-ID` for our async clients so the response can match up to the request. `ETag` is probably also common. I think these days there will often be `Keep-Alive`. That's 7 common headers for requests, but I don't know if it's usually 7.

For responses I think `Content-Length` is often there. `Location` for `POST`.   The `ETag` and `Expires` headers also. We use `X-Correlation-ID` for our async request-response. I am sure there are a few others that aren't percolating right now, so say that 6 or 7 are probably common, but I think response headers are fewer than request headers.